CHANGELOG.md

```diff
@@ -1,3 +1,13 @@
+## 1.3.1 (2024-04-25)
```

Comment:
This should be `## Unreleased` until we tag it, but if we do a release immediately after this lands that's fine.

Reply:
Cool, that's my intent.